### ISSUE TITLE
Expand all cards button - new feature

### ISSUE THREAD

**Issue by MariSvirik** [patternfly/patternfly-design#1132]
In the new redesign of the Satellite, we use a lot of cards that are expandable. We are looking for a design solution to expand/collapse all cards at once to help users with dealing with the displayed information. 
Here is an [issue](https://github.com/theforeman/foreman/pull/9054) for the Host detail tab (opens by @amirfefer) with a discussion and an implementation of the feature. There is also added "remembering mechanism" of expanded/collapsed cards.

There is already [expand/collapse all rows ](https://www.patternfly.org/v4/components/table/react-demos/#expandcollapse-all) in PF4. This issue can be either viewed as a new feature or an enhancement.

**Drafts:** 

<img width="754" alt="Screenshot 2022-02-03 at 14 36 41" src="https://user-images.githubusercontent.com/51960547/152368880-08c480a7-02c8-411a-96e9-e1984c36ae07.png">
<img width="727" alt="Screenshot 2022-02-03 at 14 37 00" src="https://user-images.githubusercontent.com/51960547/152368883-d7010c92-eae8-4e77-bae0-88bfd0737e0f.png">
<img width="707" alt="Screenshot 2022-02-03 at 14 37 51" src="https://user-images.githubusercontent.com/51960547/152368885-f82e1719-a57d-4bde-8057-1364c2df0716.png">
<img width="741" alt="Screenshot 2022-02-03 at 14 38 22" src="https://user-images.githubusercontent.com/51960547/152368890-b2560d98-c9c0-4636-8019-264a995ec682.png">

@mmenestr 
@mcarrano

**Comment by mcarrano** [comment]
@MariSvirik I don't believe there is any reason this could not be implemented now using existing PatternFly components, so I think this is more of a question on whether we would establish a new design pattern for this. This might be a good topic for our next PatternFly design share to see if others have a similar use case or feedback on your design options. Would you be interested in presenting this?

cc @mmenestr

**Comment by MariSvirik** [comment]
@mcarrano I already added the topic to the agenda. I hope we can agree on a pattern for that.

**Comment by mcarrano** [comment]
@MariSvirik looks good. Do you need anything more from PatternFly or can I close this out?

**Comment by MariSvirik** [comment]
I don't. Thank you. 
Let's put it to the PF4 guidelines!
